MapboxNavigation/RouteMapViewController.swift

```diff
@@ -684,6 +684,13 @@ extension RouteMapViewController: NavigationViewDelegate {
             streetLabelLayer.lineOpacity = NSExpression(forConstantValue: 1)
             streetLabelLayer.lineWidth = NSExpression(forConstantValue: 20)
             streetLabelLayer.lineColor = NSExpression(forConstantValue: UIColor.white)
+
+            // filter out to road classes valid for motor transport
+            let roadPredicates = ["motorway", "trunk", "primary", "secondary", "tertiary", "street"].compactMap { roadClass -> NSPredicate? in
```

Comment:
It looks like several relevant classes are omitted from this list, such as `motorway_link` (freeway ramps) and `roundabout`. The full list of supported road classes is at:

https://docs.mapbox.com/vector-tiles/reference/mapbox-streets-v7/#road
https://docs.mapbox.com/vector-tiles/reference/mapbox-streets-v8/#--road---class-text

It’s also worth considering whether we should expand the list or omit the predicate altogether if the current profile is `walking` or `cycling`.

Reply:
I included the list of motor vehicle road classes that are used to style with in the `road-label` layer of our navigation map style but can add these other ones as well. I'll put in a check for the routing profile as well.